README.md

```diff
@@ -5,9 +5,14 @@ A great example repo you can fork to your heart’s content, that serves as a la
 
 ## Getting started
 
-The first thing you want to do with this repo is quite likely **fork it**.  Use the Fork button in the top-right section of the page to do this.
-
-Once you’re on your fork, you can experiment however you like with it, play with branches, issues, milestones, labels, the wiki, and more.
+The first thing you want to do with this repo is quite likely [**fork it**](https://help.github.com/articles/fork-a-repo/).  Use the Fork button in the top-right section of the page to do this.
+
+Once you’re on your fork, you can experiment however you like with it, play with 
```

Comment:
I believe that https://help.github.com/articles/viewing-branches-in-your-repository/ would be a better entry point for branch help. :wink: